tests/test_butlerUtils.py

```diff
@@ -413,6 +413,13 @@ def test_makeDefaultLatissButlerRaiseTypes(self):
             with self.assertRaises(FileNotFoundError):
                 makeDefaultLatissButler()
 
+    def test_DAF_BUTLER_REPOSITORY_INDEX_value(self):
+        # If DAF_BUTLER_REPOSITORY_INDEX is truthy then we expect it to point
+        # to an actual file
+        repoFile = os.getenv('DAF_BUTLER_REPOSITORY_INDEX')
+        if repoFile:
+            self.assertTrue(os.path.isfile(repoFile))
```

Comment:
This is not true at the IDF.  I think you need `lsst.resources.ResourcePath(repoFile).exists()`.

Reply:
Good catch, thanks!